crc-admin-helper.spec.in

```diff
@@ -57,8 +57,9 @@ install -m 0755 -vp %{gobuilddir}/src/%{goipath}/out/linux-amd64/crc-admin-helpe
 install -d %{buildroot}%{_datadir}/%{name}-redistributable/{linux,macos,windows}
 install -m 0755 -vp %{gobuilddir}/src/%{goipath}/out/linux-amd64/crc-admin-helper %{buildroot}%{_datadir}/%{name}-redistributable/linux/
 install -m 0755 -vp %{gobuilddir}/src/%{goipath}/out/windows-amd64/crc-admin-helper.exe %{buildroot}%{_datadir}/%{name}-redistributable/windows/
-install -m 0755 -vp %{gobuilddir}/src/%{goipath}/out/macos-amd64/crc-admin-helper %{buildroot}%{_datadir}/%{name}-redistributable/macos/
-install -m 0755 -vp %{gobuilddir}/src/%{goipath}/out/macos-arm64/crc-admin-helper-arm64 %{buildroot}%{_datadir}/%{name}-redistributable/macos/
+install -m 0755 -vp %{gobuilddir}/src/%{goipath}/out/macos-amd64/crc-admin-helper %{buildroot}%{_datadir}/%{name}-redistributable/macos-amd64/
+install -m 0755 -vp %{gobuilddir}/src/%{goipath}/out/macos-arm64/crc-admin-helper %{buildroot}%{_datadir}/%{name}-redistributable/macos-arm64/
```

Comment:
I don't think we want to add `macos-amd64`/`macos-arm64` to the redistributabe tarball, just the universal binary.

Reply:
If we wanted to add these dirs, they'd have to be added to ` install -d %{buildroot}%{_datadir}/%{name}-redistributable/{linux,macos,windows}` so that the directories are created. But it's better to remove them and only ship the universal binary.